.github/workflows/releaser.yml

```diff
@@ -97,4 +97,43 @@ jobs:
              release_name: ${{ steps.prepare.outputs.tag_name }}
              tag: ${{ steps.prepare.outputs.tag_name }}
              overwrite: true
-             file_glob: true
\ No newline at end of file
+             file_glob: true
+
+ # build image and publish to dockerhub
+  docker_build_push:
+    needs: [binary_publish]
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Prepare
+        id: prepare
+        run: |
+            TAG=${GITHUB_REF#refs/tags/}
+            echo ::set-output name=tag_name::${TAG}
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
Hey @obrienalaribe, could you please check if the envs are set on the repo config?

Reply:
Yeah planning to sort that out too, just tried running the image and its getting the build error `/usr/local/bin/avail-light: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /usr/local/bin/avail-light)`